.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
   "files.associations": {
     "*.css": "postcss"
   },
-  "editor.formatOnSave": false
+  "editor.formatOnSave": false,
+  "unocss.root": "../"
```

Comment:
I am not sure if this is needed. It works fine on my side without it.

Reply:
Maybe it's because my vscode hasn't restarted 😢 .